includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2918,6 +2918,34 @@ public function get_shipping_data_from_order( WC_Order $order ): array {
 		];
 	}
 
+	/**
+	 * Create the shipping data array to send to Stripe when making a purchase, based on order's billing data.
+	 *
+	 * @param WC_Order $order The order that is being paid for.
+	 * @return array          The shipping data to send to Stripe.
+	 */
+	public function get_billing_data_from_order( WC_Order $order ): array {
```

Comment:
Given the logic is not related to the gateway itself, did we consider placing it in `WC_Payments_Order_Service`?